package.json

```diff
@@ -22,20 +22,21 @@
     "readme.md"
   ],
   "main": "dist/vue-headful.js",
+  "types": "types",
   "repository": "github:troxler/vue-headful",
   "scripts": {
-    "dev": "(cd demo; poi)",
+    "dev": "cd demo && poi",
     "build": "babel src/ --presets babel-preset-env --out-dir dist/",
-    "prepublishOnly": "npm run build"
```

Comment:
`prepublishOnly` is considered deprecated in some npm versions.

Reply:
Do you have a resource for that? It is only available in newer versions of npm, but it should not be deprecated.